src/stringify.js

```diff
@@ -124,7 +123,35 @@ var BASE_16 = 16;
         var mind = gap;
         var partial;
         var value = holder[key];
-        var isInt64 = value !== null && value instanceof Int64;
+        var isInt64 = true;
+        var Int64Keys = ["buffer", "offset"];
```

Comment:
these two variables seem to be parallel collections - shouldn't they be a constant defined as a dict (to mimic a _struct_), with two fields (keys and types)